addons/azureSets/azuresets.lua

```diff
@@ -94,8 +104,7 @@ function set_spells_from_spellset(spellset, setPhase)
 
                 windower.ffxi.remove_blue_magic_spell(slotToRemove)
                 --log('Removed spell: '..v..' at #'..slotToRemove)
```

Comment:
This looks like a previous debug statement and could also be removed, if you feel like it.